src/Fixer/Phpdoc/PhpdocLaravelOrderFixer.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\Phpdoc;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\DocBlock\DocBlock;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\FixerDefinition\FixerDefinitionInterface;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+use SplFileInfo;
+
+/**
+ * @author Julius Kiekbusch <jubeki99@gmail.com>
+ */
+class PhpdocLaravelOrderFixer extends AbstractFixer
+{
+    /**
+     * @inheritdoc
+     */
+    public function getName(): string
+    {
+        return 'phpdoc_laravel_order';
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function isCandidate(Tokens $tokens): bool
+    {
+        return $tokens->isTokenKindFound(T_DOC_COMMENT);
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function getDefinition(): FixerDefinitionInterface
+    {
+        return new FixerDefinition(
+            'Annotations must respect the following order: @param, @return, and @throws.',
```

Comment:
I believe we should rather generalise `phpdoc_order` instead of making framework-specific rule.

Reply:
Thank you for your comment.
Please tell me what to do with what. I don't know what these 3 classes do.
It is a copy-paste job from Laravel Pint. Thank you!

Reply:
I see! PhpdocOrderFixer: param -> throws -> return
PhpdocLaravelOrderFixer: param -> return -> throws

What do you mean by "generalise"?

Reply:
In general, when we have a which to do almost the same, but with a slight different outcome, to match different users expectations, we have one rule (single class), that we mark configurable. You may grep for ConfigurableFixerInterface to see examples.

Yet, if you are not original author of those fixers and you took it from another repo, I would suggest to incorporate original creator of it, to not violate his coderights/license.

I guess (I didn't checked deeply) that you copied them from laravel/pint repo, and if it is the case, I would prefer to wait for outcome of https://github.com/laravel/pint/issues/50 before we move forward with it.

Reply:
Okay.

Reply:
Suggestion:
- let's have smaller PRs that are focusing on single job only. I would suggest to simplify this PR to focus on one rule only, and then move to other rule when first one is merged
- let's introduce configuration option to the fixer rather than duplicating it
- we gonna need some new utests to cover new functionality

Reply:
It would be to nice if a developer would jump in - I mostly do the thinking.

Reply:
If you have someone from Laravel community to help, please let them know (i guess it will be them who would benefit from changes).
If not really, and you are not the one who can work on this PR, then I suggest to close it until someone eager to do it would arrive.

Reply:
https://github.com/laravel/pint/issues/47
https://github.com/laravel/pint/issues/29#issuecomment-1165280379